contrib/linux/patches-5.10/0001-switchdev-add-fid-sid-state-to-VLAN-notification.patch

```diff
@@ -0,0 +1,117 @@
+From 27dd8764c2c9ada4d9de83ae8d9d242418286b11 Mon Sep 17 00:00:00 2001
```

Comment:
curios: these patches are meant as reference for others?
we usually didn't keep kernel patches in mstpd;